src/resin/auth.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+module.exports = function(opt) {
+    return {
+        describe: function() {
+        const auth = resin.auth;
```

Comment:
Given we're only using this in one place, just using `resin.auth` should be fine.